src/assets/scripts/open-windows.bat

```diff
@@ -1,5 +1,5 @@
 @ECHO OFF
-      C:\Users\DCHAU\Documents\GitHub\workspace\dist\assets\scripts\nircmd.exe exec show "C:\Program Files\Mozilla Firefox\firefox.exe" /n
-    C:\Users\DCHAU\Documents\GitHub\workspace\dist\assets\scripts\nircmd.exe wait 500
-    C:\Users\DCHAU\Documents\GitHub\workspace\dist\assets\scripts\nircmd.exe win setsize foreground  480, 0, 1440, 270
+    start "" %1 /n
+    C:\Users\%USERNAME%\Documents\GitHub\workspace\src\assets\scripts\nircmd.exe cmdwait 1500
+    C:\Users\%USERNAME%\Documents\GitHub\workspace\src\assets\scripts\nircmd.exe win setsize foreground  %2, %3, %4, %5
```

Comment:
I don't think its safe to assume that everyone will have a filepath structured like this to the nircmd.exe. Better to handle this in the code.